### DISCUSSION TITLE
Can the  stable-diffusion-webui-depthmap-script be ported over?

### DISCUSSION THREAD

**Discussion by lapolicek** [WASasquatch/was-node-suite-comfyui#215] [Ideas]
Primarily looking for the stereo portion of the tool. Since there are 3 midas tools already, some of the work might already be done.

https://github.com/thygate/stable-diffusion-webui-depthmap-script

**Comment by WASasquatch** [comment]
I had a stereo 3D transform node that worked off image and depth input, but it had stupid requirements that I couldn't get to build properly on my system, only on colab (linux). I'll have to look into it again.

**Reply by lapolicek** [reply]
Thanks for looking into it!
